requirements/pytorch/loggers.info

```diff
@@ -1,5 +1,5 @@
 # all supported loggers. this list is here as a reference, but they are not installed in CI
-neptune>=1.0.0
+neptune
```

Comment:
>= 0.16.16 for it to support `.append` and `stringify_unsupported`

Reply:
I think this should be non-issue as `neptune` package is available only from 1.0 onwards, so both should be available.

Ref: https://pypi.org/project/neptune/#history